scripts/dev/misc_checks.py

```diff
@@ -175,10 +175,15 @@ def check_spelling(args: argparse.Namespace) -> Optional[bool]:
             "Common misspelling or non-US spelling"
         ) for w in words
     ]
+
+    qtbot_methods = {'keyClick', 'keyClicks', 'mousePress', 'keyPress'}
```

Comment:
I think we should have all methods here, even if we don't use them all.

```suggestion
    qtbot_methods = {
        'keyPress',
        'keyRelease',
        'keyClick',
        'keyClicks',
        'keyEvent',
        'mousePress',
        'mouseRelease',
        'mouseClick',
        'mouseMove',
        'mouseDClick',
        'keySequence'
    }
```